docs/crio.8.md

```diff
@@ -114,6 +115,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--apparmor-profile**="": Name of the apparmor profile to be used as the runtime's default. This only takes effect if the user does not specify a profile via the Kubernetes Pod's metadata annotation. (default: crio-default)
 
+**--big-files-temporary-dir**="": Temporary directory to use for storing big files
```

Comment:
```suggestion
**--big-files-temporary-dir**="": Path to the temporary directory to use for storing big files.
```
to match: https://github.com/cri-o/cri-o/pull/3935/files#diff-df24ddfad03f6eba45333a6630322058R275

At the very least, you need an ending period (`.`)

Reply:
Thanks for the remark!